lib/msgpax/packer.ex

```diff
@@ -103,6 +103,14 @@ defprotocol Msgpax.Packer do
   It returns an iodata result.
   """
   def pack(term)
+
+  Kernel.def(pack_nan(), do: <<0xCB, -1::64>>)
```

Comment:
Shall we write some docs for these 2 packers as well?

Reply:
Yes. Actually, I'll move packing addition to another PR.